tools/itsg33-to-gh-issues/script.py

```diff
@@ -1,31 +1,229 @@
+    if row[CONTROL_CLASS]:
+        body += "# Class\n{}\n\n".format(row[CONTROL_CLASS])
+    if row[SUPPLEMENTAL_GUIDANCE]:
+        body += "# Supplemental Guidance\n{}\n\n".format(row[SUPPLEMENTAL_GUIDANCE])
+    if row[REFERENCES]:
+        body += "# References\n{}\n\n".format(row[REFERENCES])
+    if row[GENERAL_GUIDE]:
+        body += "# General Guide\n{}\n\n".format(row[GENERAL_GUIDE])
+    if row[SUGGESTED_PLACEHOLDER_VALUES]:
+        body += "# Suggested Placeholder Values\n{}\n\n".format(row[SUGGESTED_PLACEHOLDER_VALUES])
+    if row[PROFILE_SPECIFIC_NOTES]:
+        body += "# Profile Specific Notes\n{}\n\n".format(row[PROFILE_SPECIFIC_NOTES])
+    if get_suggested_assignment(row):
+        body += "# Suggested Assignment\n{}\n\n".format(get_suggested_assignment(row))
+    if get_support_teams(row):
+        body += "# Support Teams\n{}\n\n".format(get_support_teams(row))
+    return body
+
+
+def get_labels(row):
+    """
+    Get labels for issue to help future retrieval
+    """
+    labels = []
+    if row[SUGGESTED_PRIORITY]:
+        labels.append("Priority: {}".format(row[SUGGESTED_PRIORITY]))
+    if row[CONTROL_CLASS]:
+        labels.append("Class: {}".format(row[CONTROL_CLASS]))
+    if row[ENHANCEMENT]:
+        labels.append("Control: {}-{}({})".format(row[FAMILY], row[CONTROL_ID], row[ENHANCEMENT]))
+    else:
+        labels.append("Control: {}-{}".format(row[FAMILY], row[CONTROL_ID]))
+    if get_suggested_assignment(row):
+        labels.append("Suggested Assignment: {}".format(get_suggested_assignment(row)))
+    return labels
+
+
+def get_suggested_assignment(row):
+    """
+    Get suggested assignment for issue by looking up in the fields who has the "R" (Responsible)
+    """
+    for i in range(8, 14):
+        if row[i] == "R":
+            return LABELS[i]
+
+
+def get_support_teams(row):
+    """
+    Get support teams for issue by looking up in the fields who has the "S" (Support)
+    """
+    teams = []
+    for i in range(8, 14):
+        if row[i] == "S":
+            teams.append(LABELS[i])
+    return ", ".join(teams)
+
+
+    
+
+def get_title(row):
+    """
+    Get title for issue. The logic required is encapsulated in this function. Some controls have enhancements, and if they do,
+    the title should be formatted to show such info.
+    """
+    title = ""
+    if row[ENHANCEMENT]:
+        title = "{}-{}({}): {}".format(row[FAMILY], row[CONTROL_ID], row[ENHANCEMENT], string.capwords(row[CONTROL_NAME]))
+    else:
+        title = "{}-{}: {}".format(row[FAMILY], row[CONTROL_ID], string.capwords(row[CONTROL_NAME]))
+    return title
+    
+
+def get_controls():
+    """
+    Get controls from csv file and jumps the header.
+    """
+    rows = []
+    with open(CSV_FILE, 'r') as file:
+        reader = csv.reader(file)
+        
+        if next(reader)[0] != "Family":
+            raise Exception("Headers different than expected")
```

Comment:
Just a comment that you could raise a more specific exception here like `ValueError` (or maybe there's an even better one).  The `Exception` base class is just very broad.  It doesn't matter for your case, but if this was a more complex program, it would give the caller a chance to react differently depending on the error.

Reply:
I've changed to ValueError but didn't actually find one that would fit better. We could create a specific one but for this app purpose, I think it is fine to leave that for now.